### ISSUE TITLE
[Android] seek to specific time doesn't work

### ISSUE THREAD

**Issue by bulkinav** [react-native-community/react-native-video#2915]
# Bug

On Android seeking time via useEffect doesn't work - the video always starts from the beginning (but works on iOS).
At the same time it works also on Android with v6.0.0-alpha.4

I use this behavior in the "Continue Watching" function so that the user can start the video from where he previously stopped.

## Platform

Android (using Pixel 5a with Android 13)

## Environment info

```
System:
    OS: macOS 13.0
    CPU: (6) x64 Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz
    Memory: 20.76 MB / 24.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 18.11.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 8.5.5 - /usr/local/bin/npm
    Watchman: 2022.10.31.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.1, iOS 16.1, macOS 13.0, tvOS 16.1, watchOS 9.1
    Android SDK:
      API Levels: 30, 31, 32
      Build Tools: 30.0.2, 30.0.3
      System Images: android-22 | Google APIs Intel x86 Atom_64, android-28 | Android TV Intel x86 Atom, android-30 | Google TV Intel x86 Atom, android-30 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: Dolphin 2021.3.1 Patch 1 Dolphin 2021.3.1 Patch 1
    Xcode: 14.1/14B47b - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_312 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0
    react-native: 0.70.4 => 0.70.4
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: latest from master branch.
Uses the following in package.json:

```
"react-native-video": "react-native-video/react-native-video.git",
```

## Steps To Reproduce

1. Run the video
2. Seek to some position -> save the view time through its implementation in the app
3. Run this again video -> video starts from the beginning

## Expected behaviour

1. Video start from viewed time

## Reproducible sample code

```
import Video from 'react-native-video'

export const VideoScreen = React.memo(() => {
    const appParams = useSelector(state => state.app.params)

    const route = useRoute()
    const [viewTime, setViewTime] = useState(0)

    useEffect(() => {
            if (route?.params?.viewTime && videoRef?.current) {
                videoRef?.current?.seek(route?.params?.viewTime)
            }
        }, [route?.params?.viewTime])

    useEffect(() => {
            const subscription = AppState.addEventListener('change', nextAppState => {
                appState.current = nextAppState

                if (appState.current === 'inactive') {
                    handleSendViewTime()
                }
            })
    return () => subscription.remove()
    }, 

    return (
            <View style={styles.container}>
                <Video
                    ref={videoRef}
                    source={{ uri }}
                    ...
                />
            </View>
    )
```

## Video sample
https://bulkin.me/files/screen-20221107-190622.mp4

**Comment by freeboub** [comment]
Unfortunnately, no fix are planned on 5.2 branch... 
If it work on 6.0 what's good.
Anyway, 2 interesting points:
- I need to backport on official repo a new prop in source to be able to define startTime of content (to use bookmarks then)
- I think it would be better to delay the seek call, I think you need at least to wait onLoad event. It is not easy to know when viewTime change with the sample code

**Comment by bulkinav** [comment]
@freeboub no, it was just about version 6.x But I made the changes as you recommended and everything worked. Thanks.
